doc/man3/BIO_f_ssl.pod

```diff
@@ -271,6 +271,24 @@ a client and also echoes the request to standard output.
  BIO_flush(sbio);
  BIO_free_all(sbio);
 
+=head1 RETURN VALUES
+
+BIO_f_ssl() returns the SSL BIO method.
+
+BIO_set_ssl(), BIO_get_ssl(), BIO_set_ssl_mode(), BIO_set_ssl_renegotiate_bytes(),
+BIO_set_ssl_renegotiate_timeout() and BIO_get_num_renegotiates() return 1 on
+success or a value which is less than or equals to 0 if an error occurred.
+
+BIO_new_ssl(), BIO_new_ssl_connect() and BIO_new_buffer_ssl_connect() return
+a valid B<BIO> structure on success or B<NULL> if an error occurred.
+
+BIO_ssl_copy_session_id() returns 1 on success or 0 on error.
+
+BIO_ssl_shutdown() does not return a value.
```

Comment:
Probably we don't need to list void functions in this section

Reply:
Sure? I followed other 'RETURN VALUES' sections in other POD files and I found the void functions are also documented so I keep it consistently...

Reply:
I'm not sure we are consistent, but I suspect there are many more that aren't than are. Personally I don't see a lot of value in documenting it. It is self-evident.

Reply:
Okay, if the majority are no-documented, then I am also going to remove the void-functions in RETURN-VALUE sections